README.md

```diff
@@ -1,2 +1,22 @@
 # ppod
- 
+
```

Comment:
The README should include instructions on how to run and test locally. I would also do all those steps before pushing more commits, and before every PR going forward, to confirm the container works as expected (it wouldn't have here because of the Docker command name typo).